### PR TITLE
feat: sync postgrest version from linked project

### PR DESCRIPTION
**Pull request by sweatybridge**
## What kind of change does this PR introduce?

closes https://github.com/supabase/cli/issues/389

## What is the new behavior?

- fetch PostgREST version from linked project

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: internal/utils/tenant/postgrest.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	Description string `json:"description"`
+	Version     string `json:"version"`
+}
+
+type SwaggerResponse struct {
+	Swagger string      `json:"swagger"`
+	Info    SwaggerInfo `json:"info"`
+}
+
+func GetPostgrestVersion(ctx context.Context, projectRef string) (string, error) {
```
Comment (soedirgo) [inline internal/utils/tenant/postgrest.go:24]: Have you tested this? The PostgREST version in the OpenAPI can't be used directly as Docker image versions - it looks like `11.2.0 (c820efb)`

Comment (sweatybridge) [reply internal/utils/tenant/postgrest.go:24]: Is it causing problems for you? I prefixed with `v` on line 37

The linked project I tested was on 11.1.0 though

Comment (soedirgo) [reply internal/utils/tenant/postgrest.go:24]: Yeah, just created a new project and the `.info.version` is the one above

Comment (soedirgo) [reply internal/utils/tenant/postgrest.go:24]: > I prefixed with v

The problem is the commit hash - the whitespace and everything after needs to be omitted

Comment (sweatybridge) [reply internal/utils/tenant/postgrest.go:25]: Updated code to handle both cases.

**Review by soedirgo — APPROVED** [review]

**Pull request merged**